2025-08-13T03:27:26.412Z [ERROR]: ðŸš¨ HTTP ERROR Internal server error
{
  "statusCode": 500,
  "url": "/api/chat/sid_9770be38-83d8-4e69-93f9-5f464aba5d3f",
  "ip": "127.0.0.1",
  "stack": "AggregateError\n    at Function.AxiosError.from (/Users/kormedi/Documents/WorkPlace/bitbucket/aiga-api-server/node_modules/axios/lib/core/AxiosError.js:92:14)\n    at RedirectableRequest.handleRequestError (/Users/kormedi/Documents/WorkPlace/bitbucket/aiga-api-server/node_modules/axios/lib/adapters/http.js:620:25)\n    at RedirectableRequest.emit (node:events:518:28)\n    at eventHandlers.<computed> (/Users/kormedi/Documents/WorkPlace/bitbucket/aiga-api-server/node_modules/follow-redirects/index.js:49:24)\n    at ClientRequest.emit (node:events:518:28)\n    at emitErrorEvent (node:_http_client:101:11)\n    at Socket.socketErrorListener (node:_http_client:504:5)\n    at Socket.emit (node:events:518:28)\n    at emitErrorNT (node:internal/streams/destroy:169:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:128:3)\n    at Axios.request (/Users/kormedi/Documents/WorkPlace/bitbucket/aiga-api-server/node_modules/axios/lib/core/Axios.js:45:41)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async ChatService.create (/Users/kormedi/Documents/WorkPlace/bitbucket/aiga-api-server/src/chat/chat.service.ts:61:17)\n    at async ChatController.create (/Users/kormedi/Documents/WorkPlace/bitbucket/aiga-api-server/src/chat/chat.controller.ts:120:20)",
  "type": "http_error"
}
